documentation/src/main/asciidoc/titles/cli/cli-docinfo.html

```diff
@@ -1,4 +1,4 @@
-<link rel="canonical" href="http://infinispan.org/docs/stable/user_guide/user_guide.html">
+<link rel="canonical" href="http://infinispan.org/docs/stable/index.html">
```

Comment:
I would change to HTTPS in all these *-docinfo.html 
`http://infinispan.org` => `https://infinispan.org`